### ISSUE TITLE
[4.0] Change role attribute value 

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/custom-elements#118]
#114 is not correct as it changes ALL role=alert to role=alertdialog

The issue that this was supposed to fix clearly states that this was not a possible solution

**Comment by Quy** [comment]
Can it be fixed like `type` attribute where if there is a role attribute specified then use it? What are the possible options for `role` other than `alert` and `alertdialog`?

**Comment by brianteeman** [comment]
To me the only way to "fix it" would be
1. revert this change as its clearly wrong
2. create a new ce to use when the role is alertdialog
3. update the code in joomla to use the ce for either alert or alertdialog as appropriate

I guess there might be a way to just have one ce and to pass the role from joomla to the ce and then have it conditionally displayed but thats probably just as much work.

Basically the difference between role=alert and role=alertdialog is that alertdialog can receive a response from the user

Probably should point out (again) that even without this the ce is not accessible and is misusing the role=alert

**Comment by Quy** [comment]
JS is not my forte, however, I was able to modify it to check for the `role` attribute. If absent, default to `alert`, otherwise, use the value provided that can only be `alert` and `alertdialog`. I will submit a PR in the meantime until if/when a better PR is submitted.

**Comment by wilsonge** [comment]
This has now been merged and released on npm. from the perspective of this repo done (just needs to be ported into the cms for use)
